Fix unexpected access during disposing FileStoreFixture (#150)
planetarium/libplanet: CHANGES.md

    `Peer.Urls` was renamed to `Peer.EndPoint` and its type also was changed
    from `IImmutableList<Uri>` to `IPEndPoint`.
    [[#120], [#123] by Yang Chun Ung, [#126], [#127]]

>  -  `Swarm` now ignores the blocks that are same index as tip it have, but

Reviewer: ```suggestion
 -  `Swarm` became to ignore tip blocks of the same height (`Index`) that it
```

>     processes only the longer blocks.

Reviewer: ```suggestion
    already has and deal with only longer (higher) blocks.
```
